packages/openapi-request-coercer/index.ts

```diff
@@ -283,6 +287,12 @@ function getCoercer(type, strict) {
   if (!strategy) {
     strategy = COERCION_STRATEGIES[type];
   }
+  if (strategy === undefined) {
+    // This doesn't deserve to throw an exception, by maybe logging would be
```

Comment:
*but* maybe logging would be ...

yes, please use the logger interface.  i believe that should be passed in here.

Reply:
@jsdevel Ok. I'll mimic how it's done in other packages and will extend the ctor args to accept a Logger.

Reply:
@jsdevel I've pushed a proposal.